boards/arm/nucleo_l011k4/doc/index.rst

```diff
@@ -0,0 +1,170 @@
+.. _nucleo_l011k4_board:
+
+ST Nucleo L011K4
+################
+
+Overview
+********
+The STM32 Nucleo-32 development board with STM32L011K4 MCU, supports Arduino and ST morpho connectivity.
```

Comment:
ST morpho is not supported
Supported Arduino connector is Nano V3

Reply:
Reference to ST morpho removed, and Arduino changed to "Arduino Nano V3"